packages/validator/src/cli-validator/utils/config-file-schema.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2017 - 2023 IBM Corporation.
+ * SPDX-License-Identifier: Apache2.0
+ */
+
+module.exports = {
+  title: 'ValidatorConfigSchema',
+  description:
+    'JSON schema for the user configuration file supported by the IBM OpenAPI Validator.',
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    colorizeOutput: {
```

Comment:
I'm wondering if we should be consistent with our JSON artifacts (i.e. config input and JSON output) in terms of case convention. I used snake case for the JSON output but here we have camel case. I'm not sure they need to be the same or that one should necessarily be preferred above the other (though camel case is probably best). What do you think? If we want them to be consistent and camel case, I can open a follow on PR to change the JSON structure/schema

Reply:
I think I sub-consciously associate camel-case with javascript-ish things in some vague sense so it looks "better" to me to see camel-case in the config file.  

If we were to convert the output json structure to camel-case I think we'd only need to change one property (generalized_message) so that change would be a pretty small impact.

I can include that in my PR if you agree we should do it.

Reply:
I agree, so that sounds good to me!

Reply:
made the changes to convert the results output to use camel-case